### ISSUE TITLE
parameters in `_common.R` vs. `code/params.R`

### ISSUE THREAD

**Issue by benkeser** [CoVPN/correlates_reporting#83]
@KenLi93 @cyu-hvtn @yiwenlu @youyifong @nhejazi @thebioengineer 

Currently, there is redundancy the definition of certain variables across [`_common.R`](https://github.com/CoVPN/correlates_reporting/blob/master/_common.R), which is sourced by all scripts, and in subdirectory specific folders (e.g., [`cor_coxph/code/params.R`](https://github.com/CoVPN/correlates_reporting/blob/master/cor_coxph/code/params.R)). 

To make the code bank more robust, we would like to remove this redundancy. The idea is to have `_common.R` define parameters that span across multiple analyses and reserve `code/params.R` for subdirectory-level-analysis-specific parameters. 

For example, what should be in `_common.R`?
- names of `assays`
- labels for variables
- LLOQ/LLOD (these are all over the place right now across the scripts)
- times of the analysis (D0 vs. 29 vs. 57)
- etc...

Basically all the things that were previously coming from the `COVIDcorr` package.

What should be in `params.R`? Anything specific to *your* analysis that may change across company runs; for example,
- number of bootstrap replications
- limits of axes for plots
- table options
- etc...

If something is defined in `_common.R`, it __should not__ be re-defined in your `params.R` script. This will save us headache later. 

Could you please ensure that you are able to build your subdirectory-specific analysis after modifying any `params.R` file as necessary to remove redundancies with `_common.R`?

**Comment by benkeser** [comment]
Will address once and for all in #159

**Comment by benkeser** [comment]
Addressed to greatest extent possible in #159
